### PR TITLE
Locales support for qb-phone

### PR DESCRIPTION
**Pull request by wanderrer**
**Describe Pull request**
Locales support for the qb-phone resource (made only for the scripts not for the UI)

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] yes
- Does your code fit the style guidelines? [yes/no] yes
- Does your PR fit the contribution guidelines? [yes/no] yes

### CODE REVIEW

**Review by Getpwnedbydolbajoba — APPROVED** [review]
Seems good, thanks for pr.

File: html/css/main.css [hunk 1]
```diff
@@ -315,14 +315,14 @@ body {
 
 .phone-notification-container {
     position: absolute;
-    height: 5.5vh;
+    height: 7.0vh;
```
Comment (tom-osborne) [inline html/css/main.css:318]: @wanderrer what's the reason for the css edits? Were they to fix something?

Comment (charj) [reply html/css/main.css:318]: Changing this css is outside of the scope of this PR

File: html/css/main.css [hunk 2]
```diff
@@ -315,14 +315,14 @@ body {
...
     width: 22vh;
     background-color: rgba(39, 39, 39, 0.9);
     margin: 0 auto;
     left: 0;
     right: 0;
     top: -8%;
-    border-radius: 3vh;
+    border-radius: 0.5vh;
```
Comment (charj) [inline html/css/main.css:325]: Changing this css is outside of the scope of this PR

File: server/main.lua [hunk 3]
```diff
@@ -1065,18 +1065,18 @@ QBCore.Commands.Add('bill', 'Bill A Player', {{name = 'id', help = 'Player ID'},
                         {billed.PlayerData.citizenid, amount, biller.PlayerData.job.name,
                          biller.PlayerData.charinfo.firstname, biller.PlayerData.citizenid})
                     TriggerClientEvent('qb-phone:RefreshPhone', billed.PlayerData.source)
-                    TriggerClientEvent('QBCore:Notify', source, 'Invoice Successfully Sent', 'success')
-                    TriggerClientEvent('QBCore:Notify', billed.PlayerData.source, 'New Invoice Received')
+                    TriggerClientEvent('QBCore:Notify', source, Lang:t("notify.invoice_sent"), 'success')
+                    TriggerClientEvent('QBCore:Notify', billed.PlayerData.source, Lang:t("notify.invoice_received"))
                 else
-                    TriggerClientEvent('QBCore:Notify', source, 'Must Be A Valid Amount Above 0', 'error')
+                    TriggerClientEvent('QBCore:Notify', source, Lang:t("notify.amont_abovez"), 'error')
```
Comment (charj) [inline server/main.lua:1071]: type in `amont`

**Review by TheBlokker — APPROVED** [review]
Thanks
